engine/src/flutter/runtime/runtime_controller.h

```diff
@@ -641,6 +641,12 @@ class RuntimeController : public PlatformConfigurationClient,
   // |PlatformConfigurationClient|
   std::shared_ptr<const fml::Mapping> GetPersistentIsolateData() override;
 
+  // |PlatformConfigurationClient|
+  void UpdateSemantics(SemanticsUpdate* update) override;
```

Comment:
I know you're just moving this, and I feel like I've looked at this once before, in case you you looked into it when you were writing this patch: is there a reason we couldn't declare the parameter as `const SemanticsUpdate*` in the interface?

Reply:
I don't think the method will call SemanticsUpdate::takeNodes which will take the ownership of the unordered_map

Reply:
Ah yeah that sounds familiar. Thanks!